packages/core/src/Stores/client-store.js

```diff
@@ -733,6 +733,11 @@ export default class ClientStore extends BaseStore {
         return this.isDxtradeAllowed(this.landing_companies);
     }
 
+    @computed
+    get is_bot_allowed() {
+        return this.isBotAllowed();
+    }
+
```

Comment:
is it possible to use is_bot_allowed on `app-store.js` lines `125` and `181` too?

Reply:
I don't think it is feasible as in both cases that you've mentioned, we're looking at logical OR condition between `is_mf` and `is_options_blocked` variables but in `is_bot_allowed` `is_virtual` and `is_multipliers_only` are also added into the condition!